src/seedu/addressbook/data/person/Address.java

```diff
@@ -57,3 +67,35 @@ public boolean isPrivate() {
         return isPrivate;
     }
 }
+
+class Block{
```

Comment:
Try not use inner classes. You would not want to do this at least for this module. Also comments are required for new classes.